test/libraries/SignMessageLib.spec.ts

```diff
@@ -43,5 +43,59 @@ describe("SignMessageLib", async () => {
                 await safe.signedMessages(messageHash)
             ).to.be.eq(1)
         })
+
+        it('can be used only via DELEGATE call opcode', async () => {
+          const { safe, lib } = await setupTests()
+          const messageHash = calculateSafeMessageHash(safe, "0xbaddad", await chainId())
+          
+          expect(
+              await safe.signedMessages(messageHash)
+          ).to.be.eq(0)
+
+          await expect(
+            executeContractCallWithSigners(safe, lib, "signMessage", ["0xbaddad"], [user1, user2])
+          ).to.revertedWith("GS013")
+      })
+
+        it("changes the expect storage slot without touching other ones", async () => {
```

Comment:
As mentioned before in slack, it is not really possible to test that no other storage slot was touched. I would leave this as is, but we should be aware that this test doesn't really check that no other storage slot was tested.